### DISCUSSION TITLE
lua error message when rendering blog post

### DISCUSSION THREAD

**Discussion by werkstattcodes** [quarto-dev/quarto-cli#3761] [Q&A] [answered]
Hello,

I am moving some posts from my distill blog to quarto. In several instances, I get the following error message when rendering the posts with quarto:

```
Error running filter C:/Users/Roland/AppData/Local/Programs/Quarto/share/filters/layout/layout.lua:
...ata\Local\Programs\Quarto\share\pandoc\datadir\_json.lua:167: bad 'for' initial value (number expected, got nil)
stack traceback:
        ...ata\Local\Programs\Quarto\share\pandoc\datadir\_json.lua:381: in function '_json.decode'
        ...ta/Local/Programs/Quarto/share/filters/layout/layout.lua:3754: in function 'parseLayoutWidths'
        ...ta/Local/Programs/Quarto/share/filters/layout/layout.lua:4419: in function 'layoutCells'
        ...ta/Local/Programs/Quarto/share/filters/layout/layout.lua:4240: in function <...ta/Local/Programs/Quarto/share/filters/layout/layout.lua:4233>
ERROR: unexpected character 'l' at line 1 col 1
```

Any idea what this could be due to?  I can run all cells (Quarto: Run All Cells) without the problem. The code of one such post would be available [here](https://github.com/werkstattcodes/distill_clean/blob/master/_post/2020-03-02-primary-schools-in-vienna/primary-schools-in-vienna.Rmd).

Any hint most welcome. Many thanks!

PS: I think I saw somewhere a function which allows  rendering a document only up to a specific point (in contrast to the entire document). If anyone knows what I mean, it would help me to isolate the issue to i.e. a specific chunk. Thanks again.

**Comment by cscheid** [comment, marked as answer]
Would you mind trying to isolate which part of your document causes that issue? It's hard for us to do it because the document you share has a number of data files.

**Reply by werkstattcodes** [reply]
Solved the problem. Keeping 'layout' in the chunk header instead of changing it to 'column:' triggered the error. Many thanks nevertheless for your reply.

**Reply by drarunmitra** [reply]
> Solved the problem. Keeping 'layout' in the chunk header instead of changing it to 'column:' triggered the error. Many thanks nevertheless for your reply.

Thanks for this. I have been scratching my head for ages trying to figure this out! You are a genius!
